packages/react-admin-core/src/stack/Stack.sc.tsx

```diff
@@ -1,6 +1,3 @@
 import styled from "styled-components";
```

Comment:
Das File braucht man so (ohne style-definitions) nicht mehr.

Reply:
@rabengraph ist schon in #175 gemacht